tests/formats/test_bmtun.py

```diff
@@ -10,3 +10,26 @@
 @pytest.mark.parametrize("bmtun_path", samus_returns_data.all_files_ending_with(".bmtun"))
 def test_bmtun(samus_returns_tree, bmtun_path):
     parse_build_compare_editor(Bmtun, samus_returns_tree, bmtun_path)
+
+
+@pytest.fixture()
+def bmtun(samus_returns_tree) -> Bmtun:
+    return samus_returns_tree.get_parsed_asset("system/tunables/tunables.bmtun", type_hint=Bmtun)
+
+
+def test_get_tunable(bmtun):
+    assert bmtun.get_tunable("Amiibo|CTunableReserveTanks", "fLifeTankSize") == 299.0
+
+    with pytest.raises(KeyError):
+        bmtun.get_tunable("CTunableAbilityScanningPulse", "iRange")
+
+
+def test_set_tunable(bmtun):
+    bmtun.set_tunable("Amiibo|CTunableReserveTanks", "fLifeTankSize", 199.0)
+    assert bmtun.get_tunable("Amiibo|CTunableReserveTanks", "fLifeTankSize") == 199.0
+
+    bmtun.set_tunable("CTunableMissile", "sDamageSource", "BOMB")
+    assert bmtun.get_tunable("CTunableMissile", "sDamageSource") == "BOMB"
+
+    with pytest.raises(KeyError):
+        bmtun.set_tunable("Amiibo|CTunableReserveTanks", "fPBTankSize", 10.0)
```

Comment:
this is only testing one case. use the optional match argument in raises to test both:
```py
with pytest.raises(KeyError, "Unknown tunable name: FakeTunable!"):
    bmtun.set_tunable("Amiibo|CTunableReserveTanks", "FakeTunable", 10.0)
with pytest.raises(KeyError, "Unknown class name: FakeClass!"):
    bmtun.set_tunable("FakeClass", "FakeTunable", 10.0)
```